indico/modules/events/editing/templates/editing.html

```diff
@@ -5,5 +5,22 @@
 {% endblock %}
 
 {% block content %}
-    <div class="page-content"></div>
+    <div class="page-content">
+        <div class="action-box">
+            <div class="section">
+                <span class="icon icon-tag" style="align-self: center;"></span>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Editing tags{% endtrans %}
+                    </div>
+                    {% trans %}Configure editable tags{% endtrans %}
```

Comment:
I don't think we need to mention "editable" and "editing" here. We are in the management areas of the Editing module, so it's obvious.